src/__tests__/testScreen.py

```diff
@@ -32,6 +32,10 @@
 }, {
     'name': 'selectDownSelectInverse',
     'inputs': ['f', 'j', 'f', 'A'],
+}, {
+    'name': 'selectTwoCommandMode',
+    'inputs': ['f', 'j', 'f', 'c'],
+    'pastScreen': 1
```

Comment:
here we denote we want the second screen we saw -- aka the screen we saw once we entered command mode